### PR TITLE
Upgrade to latest clean app w/ Decidim 0.27.8

### PR DESCRIPTION
**Pull request by tramuntanal**
- Upgrade Decidim to 0.27.8
- Downgrades Ruby to 3.0.7
- Installs `decidim-templates`
- Installs `decidim-cdtb`
- Upgrades `decidim-file_authorization_handler`

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -53,6 +53,7 @@
 
   # Use a real queuing backend for Active Job (and separate queues per environment)
   config.active_job.queue_adapter = :delayed_job
+  # config.active_job.queue_name_prefix = "decidim_clean_app_#{Rails.env}"
```
Comment (laurajaime) [inline config/environments/production.rb:56]: is required?

Comment (tramuntanal) [reply config/environments/production.rb:56]: no

File: public/sw.js.br
Comment (laurajaime) [inline public/sw.js.br:1]: this files must be ignored in .gitignore

**Comment by tramuntanal** [comment]
done @laurajaime

**Review by laurajaime — APPROVED** [review]

**Pull request merged**